cloud-sql/mysql/client-side-encryption/tests/test_query_and_decrypt_data.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 Google LLC
```

Comment:
same as other test - please move this to the `snippets/` directory and rename it to `query_and_decrypt_data_test.py`